packages/application-generic/src/services/feature-flags.service.ts

```diff
@@ -76,6 +76,25 @@ export class FeatureFlagsService {
     )) satisfies T;
   }
 
+  /**
+   * When we want to retrieve a global feature flag that shouldn't be dependant on any context
+   * we will use this functionality. Helpful for setting feature flags that discriminate
+   * the Novu Cloud service offerings with the self hosted users.
+   */
+  public async getGlobal<T>(
```

Comment:
Functionality to use when the feature flag to apply is general and doesn't need targeting by context (like infra needs).

Reply:
Do you mean kill system switch?

Reply:
Not sure what you mean, but for features that for any usage they just need to be on or off no matter of the user, org and env. Like our current feature flag system works.

Reply:
maybe kill system switch is not a good example, but I got what you mean